src/components/pinning-manager/pinning-manager-service-modal/PinningManagerServiceModal.js

```diff
@@ -67,8 +79,8 @@ const PinningManagerCustomModal = ({ t, onLeave, className, tReady, ...props })
 
           </div>
           <p className='f6'>
-            <Trans i18nKey="pinningCustomModal.description" t={t}>
-          Want to make your custom pinning service available to others?
+            <Trans i18nKey="pinningServiceModal.description" t={t}>
+          Want to make your Service pinning service available to others?
```

Comment:
```suggestion
          Want to make your custom pinning service available to others?
```
search/replace error?

Reply:
You're right! Nice catch 😓